src/hooks/query.rs

```diff
@@ -15,13 +15,21 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-use anyhow::Result;
+use anyhow::{anyhow, Result};
+use duckdb::arrow::array::RecordBatch;
 use pgrx::*;
 use std::ffi::CStr;
 use std::str::Utf8Error;
 
 use crate::duckdb::connection;
 use crate::fdw::handler::FdwHandler;
+use crate::schema::cell::*;
+
+macro_rules! fallback_warning {
+    ($msg:expr) => {
+        warning!("This query was not fully pushed down to DuckDB because DuckDB returned an error. Query times may be impacted. If you would like to see this query pushed down, please submit a request to https://github.com/paradedb/paradedb/issues with the following context:\n{}", $msg);
```

Comment:
this should say paradedb/pg_analytics instead of paradedb/paradedb

Reply:
change it in #140